testing/python/issue/test_tilelang_issue_2123.py

```
@@ -1,12 +1,20 @@
 from tilelang import tvm
 from tvm import tirx
 from tvm.tirx import op
-from tilelang.cuda.pipeline import CUDAPassPipelineBodyPrologue
 from tilelang.transform import LowerAccessPtr
 
+try:
+    from tilelang.cuda.pipeline import CUDAPassPipelineBodyPrologue
+    import tilelang.cuda._ffi_api as _cuda_ffi_api
+
+    _has_cuda_transforms = hasattr(_cuda_ffi_api, "LowerBlackwell2SM")
+except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)